Audio: Merge config default values into single source of truth

BenWiederhake opened the pull request:
In Serenity, we have many calls like `Config::read_i32("Foo"sv, "Bar"sv, "baz"sv, 0xC0FFEE)` with inline constants. This can easily lead to conflicting defaults, which already is the case(!) for a handful of places. I want to get to a point where there is exactly one single source of truth for each config value. This PR is a pilot to determine where exactly we want to go, and to avoid conflicting defaults.

The simplest way to store a single source of truth seems in my eyes to be a single `static constexpr` like in this PR, or in those places where it's really necessary, a `FooSettings/Defaults.h` file that is shared by `Foo` and `FooSettings`.

This PR:
- Merges two accesses to the AudioApplet>Applet>ShowPercent into one.
- Instead of having the default value somewhere as a literal in the program, store it in a `static constexpr`.

In the case of AudioApplet we can actually avoid the duplicate IPC call. This doesn't really improve performance, but it's nice.

Feel free to bikeshed the name `DEFAULT_AudioApplet_Applet_ShowPercent`. I hope the name can be done in a way where we can still parse the domain/group/key from it.

BenWiederhake commented:
Changes since last push: `DEFAULT_AudioApplet_Applet_ShowPercent` is probably a more reasonable name.

GMTA commented on Userland/Applets/Audio/main.cpp:
```diff
@@ -24,6 +24,8 @@
 #include <LibGfx/Palette.h>
 #include <LibMain/Main.h>
 
+static constexpr bool DEFAULT_AudioApplet_Applet_ShowPercent = false;
```
I get why you named this variable this way, but it diverges from our code style. Let's just name it `audio_applet_show_percent_default` for now.

GMTA reviewed (commented):
This change is fine, but we should probably think of a way to define defaults with `Config` just once instead of on every `::read_*` we do.

BenWiederhake commented:
Changes since last push:
- Rebased for convenience
- Change the name to `audio_applet_show_percent_default` – thanks @GMTA!

> we should probably think of a way to define defaults with Config just once

That's exactly where I'm trying to go with this: If the value is defined in only a single place, then it's a *single* source of truth. I intend to work my way through the codebase and unify all the 21 places (I counted) where a key has non-identical defaults. (Of these, about 12 places look like the value really differs; but IMHO all 21 places should be fixed to make it completely obvious. Btw, because I love stats: We have 184 config values in total, system-wide.)

GMTA commented:
> That's exactly where I'm trying to go with this: If the value is defined in only a single place, then it's a single source of truth.

I meant that ideally `Config` has an idea of all the available configuration keys and their default values. This would allow for enumeration, easy default/customized value detection, etc. It's way out of scope for this PR though :-) starting off by deduplicating/fixing the default values is a nice improvement already.
